prow/cmd/plank/main.go

```diff
@@ -61,6 +62,7 @@ func gatherOptions() options {
 	fs.StringVar(&o.jobConfigPath, "job-config-path", "", "Path to prow job configs.")
 	fs.StringVar(&o.buildCluster, "build-cluster", "", "Path to file containing a YAML-marshalled kube.Cluster object. If empty, uses the local cluster.")
 	fs.StringVar(&o.selector, "label-selector", kube.EmptySelector, "Label selector to be applied in prowjobs. See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors for constructing a label selector.")
+	fs.BoolVar(&o.report, "--report", true, "Whether or not to ignore report with githubClient")
```

Comment:
Please do not use bool flags that default to true. Choose a name that implies the default is false, for example `--skip-report`

Reply:
we'll flip it to default to false after https://github.com/kubernetes/test-infra/pull/9914 though? so start with true should be good?

Reply:
heh we just discussed this offline, I was ok with this, but **only** on the premise that this would be flipped to false by default once crier is mature? /shrug

Reply:
we also have `--dry-run=false` still >.>